### ISSUE TITLE
East relative-auto-UV is upside down

### ISSUE THREAD

**Issue by LatvianModder** [JannisX11/blockbench#222]
Windows 10, Desktop app v2.3.2

Model im creating:
![image](https://user-images.githubusercontent.com/3844385/51086570-ea76fd80-1750-11e9-87f9-fff2b61fc91d.png)

Faces that are fine:
![image](https://user-images.githubusercontent.com/3844385/51086577-f5ca2900-1750-11e9-8ef9-1ea71eab918b.png)

East face isn't:
![image](https://user-images.githubusercontent.com/3844385/51086589-0da1ad00-1751-11e9-9bdd-91e8e7c2ade7.png)

Example generated json. You can see that East has UVs wrong
```
"name": "center",
"from": [1, 1, 1],
"to": [15, 15, 15],
"faces": {
	"north": {"uv": [1, 1, 15, 15], "texture": "#skin_n", "cullface": "north"},
	"east": {"uv": [15, 15, 1, 1], "texture": "#skin_e", "cullface": "east"},
	"south": {"uv": [1, 1, 15, 15], "texture": "#skin_s", "cullface": "south"},
	"west": {"uv": [1, 1, 15, 15], "texture": "#skin_w", "cullface": "west"},
	"up": {"uv": [1, 1, 15, 15], "texture": "#skin_u", "cullface": "up"},
	"down": {"uv": [1, 1, 15, 15], "texture": "#skin_d", "cullface": "down"}
}
```

**Comment by LatvianModder** [comment]
Currently I fix it with regex replace :P
`"east": {"uv": \[([0-9\.]+), ([0-9\.]+), ([0-9\.]+), ([0-9\.]+)\]` -> `"east": {"uv": [$3, $4, $1, $2]`

**Comment by JannisX11** [comment]
Fixed in 2.4.
